### DISCUSSION TITLE
How to do ?

### DISCUSSION THREAD

**Discussion by Vince56190** [ezylang/EvalEx#470] [General]
I would like to write a calculation like this
IF ( A < 2700 , "( ( A * 0.2766 ) - 190 ) ) )" , IF ( A > 2701, "( (A  * - 0.2766 )-190 ) ) )"
did i do someting wrong ? this is to create a graph that goes up, peaks at 2700 and goes down

**Comment by RobertZenz** [comment]
The quotation marks are odd and some parantheses are off, as far as I can tell:

    IF ( A < 2700 , ( ( A * 0.2766 ) - 190 ), IF ( A > 2701, ( (A * - 0.2766 )-190 ) ) )

I didn't test this, though.

**Reply by RobertZenz** [reply]
This should be correct:

    IF ( A < 2700 , ( ( A * 0.2766 ) - 190 ), IF ( A > 2701, ( (A * - 0.2766 )-190 ), 2700 ) )

Slightly differently formatted:

```
IF(
	A < 2700,
	(A * 0.2766) - 190,
	IF(
		A > 2701,
		(A * - 0.2766) - 190,
		2700))
```
